modules/pmg_qt/pymol_qt_gui.py

```diff
@@ -1223,6 +1223,10 @@ def execapp():
     window = PyMOLQtGUI()
     window.setWindowTitle("PyMOL")
 
+    # fix linux dash icon/missing wmclass
+    QtCore.QCoreApplication.setApplicationName("PyMOL")
+    QtGui.QGuiApplication.setDesktopFileName("org.pymol.PyMOL")
```

Comment:
why not use `app`?
```py
app.setDesktopFileName("org.pymol.PyMOL")
```

Reply:
It's been some time since I've read the docs on how to fix it, if I recall correctly it was because of transient pop ups without a parent.